Makefile

```diff
@@ -50,7 +50,7 @@ validate:
 .PHONY: validate
 
 init-deploy: validate
-	$(KUBECTL) delete -n "$(NAMESPACE)" --ignore-not-found=true rs publisher
+	$(KUBECTL) delete -n "$(NAMESPACE)" --ignore-not-found=true replicaset publisher
```

Comment:
unrelated. but i am ok with merging it in this PR if you wish

Reply:
rs should even work. It didn't, maybe for some completely unrelated reason.